org.newerth.savagexr.json

```diff
@@ -0,0 +1,121 @@
+{
+	"app-id": "org.newerth.savagexr",
+	"branch": "master",
+	"runtime": "org.freedesktop.Platform", 
+	"sdk": "org.freedesktop.Sdk", 
+	"runtime-version": "1.6", 
```

Comment:
Can you test this under 18.08? Or does it need the older version. The older one is out of security support now and we're not recommending it for new apps.

Reply:
OK didn't know 18.08 existed, tried it everything built correctly but harfbuzz kept causing it to fail once run with:
  * ./savage.bin: symbol lookup error: /app/lib/libharfbuzz.so.0: undefined symbol: g_unichar_decompose

Tried the latest version of harfbuzz 1.8.8 but it still caused the same issue.

Looks like I'm stuck on runtime 1.6 :(